guiprep.pl

```diff
@@ -231,170 +235,185 @@
 my $main = MainWindow->new(
 	-title =>		'Guiprep Pre-processing Toolkit - '.$currentver,
 );
+my $int = $main->interp;
 
 # Set window size (pixels) the script TRIES to open with.
 $main->geometry($geometry);
 
 # Set the minimum window size.
-$main->minsize(qw(600 480));
+$main->minsize(qw(1024 768));
```

Comment:
@srjfoo - I suspect this is why you can't make the window smaller. I don't have any good context to know if this code change is good or bad. Worse-case we can split the difference and use 800x600 😁

Reply:
> @srjfoo - I suspect this is why you can't make the window smaller. I don't have any good context to know if this code change is good or bad. Worse-case we can split the difference and use 800x600 😁

Yes, it was. With current content, it's actually a hair wider than 800, but 800x600 works for me.